gui/wxpython/datacatalog/tree.py

```diff
@@ -682,6 +684,19 @@ def OnCreateMapset(self, event):
                    message=_("Unable to create new mapset: %s") % e,
                    showTraceback=False)
 
+    def OnCreateLocation(self, event):
+        """
+        Location wizard started
+        """
+        grassdatabase, location, user_mapset = (
+            create_location_interactively(self,
+                                          self.selected_grassdb[0].data['name'])
+        )
+        if location is not None:
+            if user_mapset != "PERNAMENT":
+                self.InsertMapset(name=user_mapset, location_node=location)
```

Comment:
What is that for? Reloading covers this.

Reply:
Removed.